.readthedocs.yaml

```
@@ -0,0 +1,18 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
```

```suggestion
build:
  os: ubuntu-24.04
  tools:
    python: "3.13"
```
Reading the [docs](https://docs.readthedocs.com/platform/stable/config-file/v2.html), we should use the latest supported environment unless there's a compatibility reason not to. 
Ubuntu 24.04 + Python 3.13 provide longer support and better future-proofing than 22.04/3.12.